finetuner-workflow/finetuner/requirements.txt

```diff
@@ -1,10 +1,8 @@
-transformers~=4.28.1
+transformers~=4.38.1
 numpy~=1.24.2
 wandb~=0.14.0
 torch==2.0.0
 psutil==5.9.4
-accelerate~=0.17.1
-tensorizer==1.1.0
-fastapi==0.85.1
-uvicorn==0.19.0
+accelerate==0.27.2
+tensorizer==2.8.1
```

Comment:
`tensorizer>=2.8.1`? Does it have to be hard coded to a specific version?

Reply:
`tensorizer>=2.8.1,<3.0.0` should be safe